packages/camera/tizen/src/camera_device.cc

```diff
@@ -188,36 +222,39 @@ void CameraDevice::Dispose() {
   }
 }
 
-int CameraDevice::GetDeviceCount() {
-  int count = 0;
+bool CameraDevice::GetDeviceCount(int &count) {
```

Comment:
It's a minor stuff.. but... Why is it changed like this? Do you plan to use the return value later?